packages/create-project/src/create-project.ts

```diff
@@ -184,43 +187,16 @@ export async function createApp({
 
   if (await tryGitInit(root, gitInit)) {
     console.log('Initialized a git repository.');
-    console.log();
   }
 
-  let cdpath: string;
-  if (path.join(originalDirectory, appName) === appPath) {
-    cdpath = appName;
-  } else {
-    cdpath = appPath;
-  }
+  const messageFile = path.join(root, '.post-create');
 
-  console.log(`${chalk.green('Success!')} Created ${chalk.bold(appName)} at ${chalk.bold(appPath + '/')}`);
-  console.log();
-  console.log('Inside that directory, you can run several commands:');
-  console.log();
-  console.log(chalk.cyan(`  ${displayedCommand} ${useYarn ? '' : 'run '}build`));
-  console.log('    Builds all the code.');
-  console.log();
-  console.log(chalk.cyan(`  ${displayedCommand} start`));
-  console.log('    Runs the built Worker.');
-  console.log();
-  console.log(chalk.cyan(`  ${displayedCommand} ${useYarn ? '' : 'run '}workflow`));
-  console.log('    Starts a Workflow.');
-  console.log();
-  console.log('To begin development, start Temporal Server:');
-  console.log();
-  console.log(chalk.cyan('  cd'), '~/path/to/temporal/docker-compose/');
-  console.log(`  ${chalk.cyan('docker-compose up')}`);
   console.log();
-  console.log(
-    chalk.dim.italic(
-      `If you haven't run Temporal Server before, visit:\nhttps://docs.temporal.io/docs/typescript/getting-started/`
-    )
-  );
+  console.log(`${chalk.green('Success!')} Created project ${chalk.bold(appName)} at:`);
   console.log();
-  console.log(`Then, in the ${chalk.bold(cdpath + '/')} directory, using two other shells, run these commands:`);
+  console.log(chalk.bold(appPath + '/'));
   console.log();
-  console.log(`  ${chalk.cyan(`${displayedCommand} ${useYarn ? '' : 'run '}start.watch`)}`);
-  console.log(`  ${chalk.cyan(`${displayedCommand} ${useYarn ? '' : 'run '}workflow`)}`);
+  await spawn('npx', ['chalk', '-t', `"$(cat ${messageFile})"`], { stdio: 'inherit' });
```

Comment:
I'd prefer reading the file contents using node and using chalk-template directly instead of spawning a process: https://github.com/chalk/chalk-template

Reply:
It's not currently published to npm, but it looks like it [might be soon](https://github.com/chalk/chalk-template/pull/3#pullrequestreview-794649035). After this is merged I could open a draft PR based on the future package?

Reply:
So until then please avoid the shell expansion.
You should be able to feed the template directly as an arg here using:
```ts
await fs.readFile(messageFile, 'utf8')
```

Reply:
Done in https://github.com/temporalio/sdk-typescript/pull/349/commits/eb64788a1becec562d54d545ff3af6166f45d4e0